### ISSUE TITLE
Not compatible with El Capitan?

### ISSUE THREAD

**Issue by motorambler** [ImageOptim/ImageOptim#160]
I just installed a fresh copy of El Capitan on my Macbook Air (2012), and also the latest version of ImageOptim. I have hundreds of varying image types on my Desktop, but every image I drag into ImageOptim, whether jpeg, png, etc., gives an error (red exclamation mark) with 0% compression. All program settings are at default. 

I have been using ImageOtpim since OSX Mavericks without issue. Is ImageOptim not compatible with El Capitan?

**Comment by pornel** [comment]
I'm developing ImageOptim using El Capitan, so it's best supported on that system.

If you're getting errors, please check if Console.app contains more details about the errors.

**Comment by motorambler** [comment]
Here are the list of steps I just did:

1. Deleted ImageOptim
2. Downloaded ImageOptim again
3. Copied ImageOptim to Applications folder
4. Opened Console and cleared the history
5. Opened ImageOptim and dragged 2 images into the app (1 x png and 1 x jpeg)

Here's the screenshot of the Console after I opened and tried to use ImageOptim:

![console](https://cloud.githubusercontent.com/assets/16861293/17981733/c14afd58-6ab9-11e6-93ef-95ee9738c56c.jpg)

**Comment by pornel** [comment]
Interesting. Looks like some files are missing from inside the app. Do you use some 3rd party app to decompress the downloaded archive?

**Comment by motorambler** [comment]
I just figured out what was happening. After I downloaded ImageOptim, I unzipped it and immediately dragged it into my Applications folder. What I didn't know is that ImageOptim was not being allowed to run. 

For some reason, OSX did **not** notify me (via warning dialog box) that "...This app was downloaded from unidentified developer bla bla bla..." when I ran the app. I only noticed by chance that ImageOptim was waiting to be allowed to run in System Preferences > Security & Privacy.

I have no idea why I didn't get the warning popup box, but am glad it's all sorted.

Thanks very much for offering your time to help solve this issue!
